api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -961,54 +961,40 @@ public DbUser getByUsernameOrThrow(String username) {
 
   @Override
   public void updateBypassTime(long userDatabaseId, AccessBypassRequest accessBypassRequest) {
-    final DbUser user =
-        getByDatabaseId(userDatabaseId)
-            .orElseThrow(
-                () ->
-                    new NotFoundException(
-                        String.format("User with database ID %d not found", userDatabaseId)));
+    getByDatabaseId(userDatabaseId)
```

Comment:
Could remove this check entirely?  I decided not to change existing behavior

Reply:
Yeah, if this user isn't found, we'll probably get an NPE (or worse) in setBypassTimeWithRetries. That would be good to verify in the debugger.